sdk/spring/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
```

Comment:
Could you help add a section for something like this (I copied this from 4.5.0-beta.1 changelog, but we need to change this for 4.5.0)

```
### Spring Cloud Azure Starter
This section includes changes in `spring-cloud-azure-starter` module.
#### Features Added
+ Support Azure hosted PostgreSQL and MySQL services authenticating with Azure AD [#30024](https://github.com/Azure/azure-sdk-for-java/pull/30024).
```

which means we are GAing these two libs.